src/FileGetContents.php

```diff
@@ -198,6 +266,8 @@ protected function getTlsStreamContextDefaults($cafile)
      * @throws \RuntimeException If https proxy required and OpenSSL uninstalled
      *
      * @return resource Default context
+     *
+     * @final since 1.1.0
      */
     protected function getMergedStreamContext($url)
```

Comment:
Assuming there's no changes in its behaviour, but sure it's probably no harm to stay close to the original source.